site/global-guides/voice/inAppGuide.mdx

```diff
@@ -91,17 +91,13 @@ The following steps can be followed to generate your credentials:
 
 ![Generating WebRTC Credentials](@site/static/img/docs-diagrams/voice/in-app-credentials.png)
 
-Once you have generated your credentials you can call our identity services endpoint. Please refer to here for more information.
+Once you have generated your credentials you can retrieve an access token from our token provider.
 
 :::note
-The scope must be **voice.webrtc** to generate an access token to use for in-app calling.
+The token **MUST** contain the scope **voice.webrtc** to use for in-app calling.
 :::
 
-As the token will be visible within your application we strongly advise to only use this scope. The token should be fetched from a server running in a secure environment, and provided to clients in a secure manner.
-
-We recommend regenerating the token every 30 seconds before expiry. It is not advisable to regenerate the token for every in-app voice call as this will create an unnecessary delay in the call being connected.
-
-Bandwidth accepts no responsibility for the loss of account credentials and any resulting network traffic, fraud, or undesired account access that results from failing to manage account access credentials in a completely secure manner. Minting of tokens is not included as a client SDK capability for the above reason.
+Please refer to the documentation on [how to retrieve an access token](./authentication.mdx) for more information.
```

Comment:
do you know if the link should work on https://gravity-1030.staging-dev.bandwidth.com/global-guides/voice/guides/inAppCalling/#generating-credentials-for-an-access-token? I receive a 404